pages/record/_.vue

```diff
@@ -80,7 +80,7 @@
         <div class="card p-3 mb-3 bg-grey">
           <MediaActionBar
             :url="selectedMedia.about"
-            :share-url="canonicalURL"
+            :share-url="canonicalUrl"
```

Comment:
Is this - and the other references in this file - still valid ? The SocialShare component no longer accepts shareUrl as a prop.